README.md

```diff
@@ -29,57 +26,43 @@ We appreciate your feedback and energy helping us improve our services. [Please
 - Create an Azure account with an active subscription. For details, see [Create an account for free](https://azure.microsoft.com/free/?WT.mc_id=A261C142F).
 - [Node.js (12.18.4 and above)](https://nodejs.org/en/download/)
 - [Visual Studio (2017 and above)](https://visualstudio.microsoft.com/vs/)
-- [.NET Core 3.1](https://dotnet.microsoft.com/download/dotnet-core/3.1) (Make sure to install version that corresponds with your visual studio instance, 32 vs 64 bit)
 - Create an Azure Communication Services resource. For details, see [Create an Azure Communication Resource](https://docs.microsoft.com/en-us/azure/communication-services/quickstarts/create-communication-resource). You'll need to record your resource **connection string** for this quickstart.
 
 ## Code structure
 
-- ./Chat/ClientApp: frontend client
-  - ./Chat/ClientApp/src
-    - ./Chat/ClientApp/src/Components : React components to help build the client app chat experience
-    - ./Chat/ClientApp/src/Containers : Connects the redux functionality to the React components
-    - ./Chat/ClientApp/src/Core : Containers a redux wrapper around the Chat SDK
-    - ./Chat/ClientApp/src/index.js : Entry point for the client app
-- ./Chat/Controllers : Server app core logic for client app to get a token to use with the Azure Communication Services Web Chat SDK
-- ./Chat/Program.cs : Server app program logic
-- ./Chat/Startup.cs: Server app startup logic
+- ./Chat/src/app: Where the client code lives
```

Comment:
nit: can we rename `Chat` to something that suggestions frontend web app?
`ChatApp`, `WebApp`, `App` ?